### PR TITLE
Home update

### PR DESCRIPTION
**Pull request by Tana97**

### CODE REVIEW

File: src/Pages/homePage/homePage.js [hunk 1]
```diff
@@ -1,8 +1,91 @@
...
+            .then((res) => res.json())
+            .then((res) => {
+                if (res.success) {
+                    setProducts(res.data);
+                    setSelectedOrderBy("");
+                }
+            });
+    }, [debouncedSearch]);
+
+    Date.prototype.addDays = function (days) {
```
Comment (rokaskasperavicius) [inline src/Pages/homePage/homePage.js:65]: This is a way to add custom things to classes. However, React is trying to move from OOP to Functional Programming. I would have probably used a simple library which can format days

File: src/Pages/homePage/homePage.js [hunk 2]
```diff
@@ -1,8 +1,91 @@
...
+            });
+    }, []);
+
+    useEffect(() => {
+        const newOrder = selectedOrderBy || "name";
+        setProducts(orderBy(products, [newOrder]));
+        // eslint-disable-next-line react-hooks/exhaustive-deps
+    }, [selectedOrderBy]);
+
+    useEffect(() => {
```
Comment (rokaskasperavicius) [inline src/Pages/homePage/homePage.js:46]: is this used on this page?

File: src/Pages/homePage/homePage.js [hunk 3]
```diff
@@ -1,8 +1,91 @@
...
+            .then((res) => res.json())
+            .then((res) => {
+                if (res.success) {
+                    // Set new categories
+                    setProducts(res.data);
+                }
+            });
+    }, []);
+
+    useEffect(() => {
```
Comment (rokaskasperavicius) [inline src/Pages/homePage/homePage.js:40]: is this used on this page?

**Review by rokaskasperavicius — APPROVED** [review]
Approved but there is some code you could clean up which is not used on the Home page

File: src/Pages/homePage/homePage.js (hunk 1 above)
Comment (Tana97) [reply src/Pages/homePage/homePage.js:65]: I will look into that

File: src/Pages/homePage/homePage.js (hunk 3 above)
Comment (Tana97) [reply src/Pages/homePage/homePage.js:40]: I will look into that

**Pull request merged**